langlib/lang.transaction/src/main/ballerina/transaction.bal

```diff
@@ -68,6 +91,19 @@ public isolated function getInfo(byte[] xid) returns Info? = @java:Method {
 
 # Prevents the global transaction from committing successfully.
 #
+#  ```ballerina
+# transaction {
+#     transaction:setRollbackOnly(notFoundError);
+#     check commit;
+# }
+#
+# transactional function transFunc() returns error? {
+#    transaction:setRollbackOnly(notFoundError);
+# }
+#
+# error notFoundError = error("not found");
```

Comment:
Does this have to be a separate error value? Can't this be constructed in-line?

Reply:
We used this error in both cases. So doesn't it give more clarification?

Reply:
How does using the same error have an impact here?

Reply:
I thought to giving a better use case 🤔